cg/store/models.py

```diff
@@ -1030,6 +1055,9 @@ class SampleRunMetrics(Base):
     run_metrics_id: Mapped[int] = mapped_column(ForeignKey("run_metrics.id"))
     type: Mapped[DeviceType]
 
+    run_metrics: Mapped[RunMetrics] = orm.relationship(back_populates="sample_metrics")
```

Comment:
we should add a cascade on delete. If a run_metrics entry gets deleted, this should also remove associated sample_run_metrics